project-koku/koku
# [COST-3216] - reset the meta data when order_by[date] is used

## Jira Ticket

[COST-3216](https://issues.redhat.com/browse/COST-3216)

## Description

* This change will ensure that the `order_by[date]` is found in the `order_by` meta, rather than in a new `cost_explorer_order_by` field.
* This change also updates the query handle `order_by` pandas functionality to handle non-unique order by categories.

## Testing

1. Go to [http://localhost:8000/api/cost-management/v1/reports/aws/costs/?order_by[date]=2022-11-08&group_by[tag:app]=*](http://localhost:8000/api/cost-management/v1/reports/aws/costs/?order_by[date]=2022-11-08&group_by[tag:app]=*)
2. See in meta:
```
        "order_by": {
            "date": "2022-11-08"
        },
```
and do NOT see `cost_explorer_order_by`.

## Notes

...
